contentcuration/contentcuration/serializers.py

```diff
@@ -75,49 +75,22 @@ def update(self, instance, validated_data):
         ret = []
         update_files = {}
         user = self.context['request'].user
+        import ipdb
         with transaction.atomic():
+        # Get files that have the same contentnode, preset, and language as the files that are now attached to this node
             for item in validated_data:
-                item.update({
-                    'preset_id': item['preset']['id'],
-                    'language_id': item.get('language')['id'] if item.get('language') else None
-                })
-
-                # User should not be able to change files without a display
-                if item['preset']['display']:
```

Comment:
long story short:

There was no need for us to be setting the file's preset in the serializer, and attempting to do so was actually interfering with saving metadata, because 'preset' was no longer being sent in items in `validated_data`.  This seems to be a remnant of some long since forgotten implementation.